assets/src/application/ui/input/Button/style.scss

```diff
@@ -0,0 +1,493 @@
+/** -------------------------------------------------------------------------
+ * Event Espresso Button Stylesheet
+ * © 2020 Event Espresso
+ * ------------------------------------------------------------------------- */
+
+:root {
+	--ee-icon-button-size: 36px;
+}
+
+.ee-event-editor {
+	.ee-button.ee-button {
```

Comment:
oooo i just realized that this won't work as soon as we add a new domain other than the event editor...

maybe using an `ee-btn-base` class for common button styles would work though because then we can target `.ee-btn-base.ee-button` which would NOT apply to any legacy buttons

Reply:
thx, good suggestion